Tests/FunctionalTests/fss-search-query.spec.ts

```diff
@@ -54,7 +58,7 @@ describe('Test Search Query Scenario On Search Page', () => {
   })
 
   it('Batch Attribute table returns correct product on special characters search', async () => {        
-   
+    await page.reload();
```

Comment:
With the proposed `beforeAll`/`beforeEach` changes, this code becomes unnecessary

Reply:
Updated the script. Thanks